spec/bundler/gem_helper_spec.rb

```diff
@@ -253,6 +254,21 @@ def remove_push_guard(gemspec_content)
 
           Rake.application["release"].invoke
         end
+
+        it "when user.signingkey is present" do
+          mock_build_message app_name, app_version
+          mock_confirm_message "Tagged v#{app_version} signed with ABCDEF01."
+          mock_confirm_message "Pushed git commits and tags."
+          expect(subject).to receive(:rubygem_push).with(app_gem_path.to_s)
+
+          Dir.chdir(app_path) { sys_exec("git push -u origin master", true) }
+
+          Dir.chdir(app_path) do
+            sys_exec("git config gpg.program #{File.expand_path("../../support/fake-gpg", __FILE__)}")
```

Comment:
Fixed in https://github.com/bundler/bundler/pull/4743/commits/aae87129f60052bb45156474295db31aea015164